source/Calamari/Integration/Nginx/NginxServer.cs

```diff
@@ -116,6 +117,10 @@ public NginxServer WithServerBindings(IEnumerable<Binding> bindings, IDictionary
             }
 
             return this;
+
+            string GetCertificateWithChainIfPresent(string certificatePem, string chainPem)
```

Comment:
I am unsure what this is actually supposed to return?

Reply:
It returns the server certificate and any chained certificates combined into a new-line separated string.

Reply:
Would `GetServerCertificateCombinedWithAnyChainedCertificates` make it easier to grok?

Reply:
Or maybe `CombineServerCertificateWithAnyChainedCertificates`

Reply:
no, what you have is good enough, I just wasn't sure what exactly it was writing out, at first I thought it was a file path.